### PR TITLE
Add object support in ToggleGen

### PR DESCRIPTION
**Pull request by ashalva**
I have finalised `object` support in Toggles. Now it's time to add `object` support in `ToggleGen`.

* Added `Object` and `ObjectSupportedType` in `ToggleGen` package.
* Executed `ToggleGen` script to generate new object properties. 
* Updated demo app to demonstrate object observable functionality.

<video width='200' src='https://github.com/TogglesPlatform/Toggles/assets/9494262/5b76f585-72ab-40ff-baee-546f884764c8'>

### CODE REVIEW

File: ToggleGen/GeneratedCode/ToggleAccessor.swift [hunk 1]
```diff
@@ -39,24 +41,31 @@ extension ToggleAccessor {
...
     }
 
-    public var userDefinedNumericToggle: Double {
+    public var numericToggle2: Double {
         get { manager.value(for: ToggleVariables.numericToggle2).numberValue! }
         set { manager.set(.number(newValue), for: ToggleVariables.numericToggle2) }
     }
 
-    public var userDefinedStringToggle: String {
+    public var stringToggle2: String {
```
Comment (ashalva) [inline ToggleGen/GeneratedCode/ToggleAccessor.swift:59]: Script automatically changed property names. Not sure why that happened, I guess it was outdated file? @albertodebortoli

Comment (albertodebortoli) [reply ToggleGen/GeneratedCode/ToggleAccessor.swift:59]: I noticed that too when running it locally yesterday. Yes, I think it was outdated.

File: ToggleGen/GeneratedCode/ToggleVariables.swift [hunk 2]
```diff
@@ -1,8 +1,10 @@
 //  ToggleVariables.swift
 
+// swiftlint:disable file_length
+
 import Foundation
 
-public struct ToggleVariables {
+public enum ToggleVariables {
```
Comment (albertodebortoli) [inline ToggleGen/GeneratedCode/ToggleVariables.swift:7]: yep, definitely outdated

File: Sources/Models/Object.swift [hunk 3]
```diff
@@ -20,7 +20,7 @@ public struct Object {
     
     /// Object description represented as plain json string. 
     /// Returns nil if object cant be represented as json string.
-    var description: String? {
+    public var description: String? {
```
Comment (albertodebortoli) [inline Sources/Models/Object.swift:23]: I remember efforts to keep this non-public but I guess it's ok (you've used it in the demo app).

**Review by albertodebortoli — APPROVED** [review]
Looking good! Approved and thanks.

This view is not showing too well aligned now

image

if it's not too much bother cold you take a look? Ok if not. Instead of having an HStack w/  2 VStacks, 1 VStack with multiple HStacks should do.

File: Sources/Models/Object.swift (hunk 3 above)
Comment (ashalva) [reply Sources/Models/Object.swift:23]: Yes, thats the thing. I could not find any better way to beutify object's presentation. I was using `objectValue.map.description` but it was showing types as well

image

**Comment by ashalva** [comment]
Nice catch @albertodebortoli, updated

**Review by albertodebortoli — APPROVED** [review]

**Review by AlexL271828 — APPROVED** [review]

**Pull request merged**